Client/src/Pages/Monitors/Home/index.jsx

```diff
@@ -24,11 +24,20 @@ const Monitors = ({ isAdmin }) => {
     dispatch(getUptimeMonitorsByTeamId(authState.authToken));
   }, [authState.authToken, dispatch]);
 
-  const up = monitorState.monitors.reduce((acc, cur) => {
-    return cur.status === true ? acc + 1 : acc;
-  }, 0);
+  const monitorStats = monitorState.monitors.reduce(
+    (acc, monitor) => {
+      if (monitor.isActive === false) {
+        acc["paused"] += 1;
+      } else if (monitor.status === true) {
```

Comment:
should we rename this to isUp instead of status?

Reply:
Yes something along those lines would make more sense, but that's a fairly large refactor.

We should open a separate issue and PR for that

Reply:
Sounds good to me!